packages/hdwallet-ledger/src/utils.ts

```diff
@@ -96,503 +95,277 @@ export const createXpub = (
     areTransactionTimestamped: true,
   },
   // 133: {
-  //   name: "zcash",
-  //   satoshi: 8,
-  //   unit: "ZEC",
-  //   apiName: "zec",
   //   bitcoinjs: {
-  //     messagePrefix: "Zcash Signed Message:",
   //     bip32: {
   //       public: {
   //         p2pkh: 76067358
   //       },
-  //       private: 87393172
   //     },
-  //     pubKeyHash: 7352,
-  //     scriptHash: 7357,
-  //     wif: 128
   //   },
   //   isSegwitSupported: false,
-  //   handleFeePerByte: false,
   //   areTransactionTimestamped: undefined,
-  //   expiryHeight: Buffer.from("00000000", "hex")
   // },
   141: {
-    name: "komodo",
-    satoshi: 8,
-    unit: "KMD",
-    apiName: "kmd",
     bitcoinjs: {
-      messagePrefix: "Komodo Signed Message:",
       bip32: {
         public: {
           p2pkh: 4193182861,
         },
-        private: 87393172,
       },
-      pubKeyHash: 60,
-      scriptHash: 85,
-      wif: 128,
     },
     isSegwitSupported: false,
-    handleFeePerByte: false,
     areTransactionTimestamped: undefined,
   },
   156: {
-    name: "bitcoin gold",
-    satoshi: 8,
-    unit: "BTG",
-    apiName: "btg",
     bitcoinjs: {
-      messagePrefix: "Bitcoin gold Signed Message:",
       bip32: {
         public: {
           p2pkh: 76067358,
           p2sh: 77429938,
         },
-        private: 76066276,
       },
-      pubKeyHash: 38,
-      scriptHash: 23,
-      wif: 128,
     },
     sigHash: 0x41,
     isSegwitSupported: true,
-    handleFeePerByte: true,
     areTransactionTimestamped: undefined,
-    additionals: ["gold"],
   },
   171: {
-    name: "hcash",
-    satoshi: 8,
-    unit: "HSR",
-    apiName: "hsr",
     bitcoinjs: {
-      messagePrefix: "HShare Signed Message:",
       bip32: {
         public: {
           p2pkh: 76071454,
         },
-        private: 87393172,
       },
-      pubKeyHash: 40,
-      scriptHash: 100,
-      wif: 128,
     },
     isSegwitSupported: false,
-    handleFeePerByte: false,
     areTransactionTimestamped: true,
   },
   121: {
-    name: "zencash",
-    satoshi: 8,
-    unit: "ZEN",
-    apiName: "zen",
     bitcoinjs: {
-      messagePrefix: "Zencash Signed Message:",
       bip32: {
         public: {
           p2pkh: 76067358,
         },
-        private: 87393172,
       },
-      pubKeyHash: 0x2089,
-      scriptHash: 0x2096,
-      wif: 128,
     },
   },
   3: {
-    name: "dogecoin",
-    satoshi: 8,
-    unit: "Ð",
-    apiName: "doge",
-    appName: "Dogecoin",
     bitcoinjs: {
-      messagePrefix: "Dogecoin Signed Message Much Wow:",
       bip32: {
         public: {
           p2pkh: 49990397,
         },
-        private: 87393172,
       },
-      pubKeyHash: 30,
-      scriptHash: 22,
-      wif: 128,
     },
   },
 };
 
+const appNameBySlip44: Record<number, string> = {
+  0: "Bitcoin",
+  2: "Litecoin",
+  3: "Dogecoin",
+  5: "Dash",
+  20: "Digibyte",
+  60: "Ethereum",
+  118: "Cosmos",
+  145: "Bitcoin Cash",
+  931: "THORChain",
+};
+
```

Comment:
THORChain and Cosmos do not require any metadata from `networksUtil`, so it made no sense to add an entry there for the sake of enabling app name lookup. Instead, I've broken it out into the lookup above.